my_modules.ts

```diff
@@ -0,0 +1,15 @@
+export function addNum(a: number, b: number): number {
```

Comment:
For the task number 8:

For this module that you've created, following typescript type declaration file could be used.

```

export declare function addNum(a: number, b: number): number;

export declare function subNum(a: number, b: number): number;

export declare class Person {
  constructor(name: string, age: number);
  name: string;
  age: number;
  sayHello(): void;
}
```

You can save the above code in a file named my_module.d.ts. This declaration file provides the type information for the functions addNum and subNum, as well as the Person class.

When using these functions or instantiating the Person class in your TypeScript code, make sure to include a reference to the declaration file at the top of your file, like this:
<br> 
`/// <reference path="my_module.d.ts" />`
<br>
 Now you can use the functions and class with type safety

```
const result = addNum(2, 3);
console.log(result); // Output: 5

const person = new Person('John Doe', 25);
person.sayHello(); // Output: Hello, I am John Doe and I am 25

```

By referencing the functions.d.ts file, TypeScript will be able to provide autocomplete suggestions and type checking for the functions and class defined in that file.